src/applications/vaos/appointment-list/components/confirmedAppointmentDetailsPage/AtlasLocation.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import FacilityAddress from '../../../components/FacilityAddress';
+import {
+  getATLASConfirmationCode,
+  getATLASLocation,
+} from '../../../services/appointment';
+
+export default function AtlasLocation({ appointment }) {
+  const { address } = getATLASLocation(appointment);
+  const { city, state } = address;
+  const confirmationCode = getATLASConfirmationCode(appointment);
+
+  const name = `ATLAS facility in ${city}, ${state}`;
+  const facility = {
+    name,
+    address,
+  };
+
+  return (
+    <div>
+      <FacilityAddress facility={facility} showDirectionsLink />
+      <div className="vads-u-font-weight--bold vads-u-margin-top--2">
```

Comment:
I think I updated this to be a heading in the old component (after you started this PR), so we may want to carry over that change.

Reply:
I see it now, fixing.